src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -913,6 +913,10 @@ private function registerProfilerConfiguration(array $config, ContainerBuilder $
 
         $container->getDefinition('profiler_listener')
             ->addArgument($config['collect_parameter']);
+
+        if (!$container->getParameter('kernel.debug')) {
```

Comment:
```suggestion
        if (!$container->getParameter('kernel.debug') || !class_exists(CliRequest::class)) {
```
at least now that `CliRequest` & co have been moved to the component?

Reply:
Done.